### ISSUE TITLE
Role=status property is missing for Status related field in Dashboard screen.

### ISSUE THREAD

**Issue by rgarg24** [percussion/percussioncms#880]
**Describe the bug**
role=status property is missing for all status related fields.

**To Reproduce**
1. Go to Dashboard screen (http://localhost:9992/cm/app/?view=dash)
2. Now, go to "Add Dashboard Gadgets" and add "Process Monitor" Dashboard gadget.
3. You will see there are two columns "Process" and "Status".
4. Inspect elements given under Status column and you will find that role=status is missing for tr and td elements.

**Expected behavior**
role=status property must be available for all status related fields.

**Additional context**
Navigate all the options available for similar issue and fix.
